src/codeu/chat/server/Server.java

```diff
@@ -176,6 +203,19 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    // Get Interests By userId - A client wants to get a subset of the interests given a userId
```

Comment:
This function, as written, only allows the client to get a single Interests object for a single user, I think it should be able to get a Collection of them to match the functionality of the other get functions, but if not, then just change the comment so it reflects the actual behavior.